xbatcher/_version.py

```diff
@@ -0,0 +1,7 @@
+from importlib.metadata import PackageNotFoundError, version
+
+try:
+    __version__ = version(__name__)
+except PackageNotFoundError:
+    # package is not installed
+    __version__ = 'unknown'
```

Comment:
Need to delete this file after 25595fbe13902e343734705d6c10bab9fa152285

Reply:
Thanks for the catch! @weiji14